assets/src/latest-stories-block/block/edit.css

```diff
@@ -273,12 +265,19 @@
   margin-right: 32px;
   min-width: 250px;
   flex-shrink: 0;
+  border: 1px solid #8080805e;
 }
 
-.wp-block-web-stories-latest-stories.is-view-type-carousel
-  .latest-stories__inner-wrapper {
-  border: 1px solid #8080805e;
+/* Stories Block Styles */
+
+.wp-block-web-stories-latest-stories.is-style-default.is-view-type-grid
+  .latest-stories__controller,
+.wp-block-web-stories-latest-stories.is-style-default.is-view-type-list
+  .latest-stories__story-placeholder,
+.wp-block-web-stories-latest-stories.is-style-default.is-view-type-carousel
+  .latest-stories__story-wrapper {
```

Comment:
Can we not use `:not` operator to exclude circles view?

Reply:
These are three different selectors, if it were to be the same then using `:not()` would've been helpful.

Reply:
My bad, didn't notice that last selectors are different 👍